templates/hooks/table/sdk_create_post_set_output.go.tpl

```diff
@@ -2,4 +2,9 @@
 		if err := rm.syncTTL(ctx, desired, &resource{ko}); err != nil {
 			return nil, err
 		}
+	}
+	// Check if replicas were specified during creation
+	// Return with a requeue to process replica updates
+	if len(desired.ko.Spec.TableReplicas) > 0 {
+		return &resource{ko}, requeueWaitWhileUpdating
```

Comment:
Should this be `requeueWaitWhileCreating` ? another note, we always do a ReadOne after a create, so probably you don't need to requeue here. Check https://github.com/aws-controllers-k8s/runtime/blob/main/pkg/runtime/reconciler.go#L661-L670